README.md

```diff
@@ -5,3 +5,13 @@ Learn how to start coding.
 Developed for people new to the background of coding.
 
 Your contributions are more than welcome.
+
+# Competitive Programming 
+Want to land a job at Google? Want to crack interviews of software tech-giants like Amazon or Microsoft? Here comes Competitive Programming as the knight in the shining armor :)
+
+![2019 ACM-ICPC World Finals at Porto, Portugal](https://miro.medium.com/max/700/1*ET6rUxY3ll6u7ohuhEXIAw.jpeg)
+
+Many of you might have heard about the term Competitive Programming and would have wondered what it means. I am writing this blog to provide a roadmap as well as a brief introduction to it. Even if you are not a computer science student, I would suggest you read this blog. The reason being it helps one develop logical aptitude and break down complex problems into simpler sub-problems. It teaches you how to deal with a problem under particular constraints and still produce an optimal solution.
```

Comment:
Can you please provide the competitive programming details about the application as well?
That is the content of the application description is missing .
Please reduce the blog description as well .

Reply:
The following changes have been done:-
1. Blog description is reduced.
2. Provided the CP details mentioned in the application.
3. Added the description of PR as well.
Let me know if I can do anything else :)